spec/glossary_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+def create_glossary_entry(glossary)
+  click_link 'Glossary'
+  click_link 'Create new entry'
+  fill_in_glossary_form(glossary.term, glossary.definition)
```

Comment:
Love this helper and specifically love the nested DRY use of `fill_in_glossary_form`